src/hotspot/share/utilities/vmError.cpp

```diff
@@ -910,7 +910,9 @@ void VMError::report(outputStream* st, bool _verbose) {
 
   STEP("printing code blobs if possible")
 
-     if (_verbose && _context) {
+     if (_verbose) {
+       frame fr = _context ? os::fetch_frame_from_context(_context)
+                           : os::current_frame();
```

Comment:
This seems like it is only need in the else branch at L931, so I'd prefer to see it there please.